[CRITICAL] main build broken: SoundCell redeclared (PR #88 missed cleanup)

## Симптом

После merge'а PR #88 (extract CreateAlarm cell subclasses) на main collision двух типов `SoundCell`:

```
Services/Alarms/Cells/SoundCell.swift:7
ViewControllers/Alarms/SoundPickerViewController.swift:92
```

Оба объявлены `final class SoundCell: UITableViewCell`. Project использует FileSystemSynchronized groups → оба подхватываются билдом → `error: invalid redeclaration of 'SoundCell'`.

## Воспроизведение

```
git checkout main
xcodebuild -scheme SnoozePay build
# → SoundCell.swift:7:13: error: invalid redeclaration of 'SoundCell'
```

## Что делать

PR #88 extract'ил `SoundCell` для CreateAlarm cells, но не убрал старый класс из `SoundPickerViewController.swift`. Варианты:

- A) Переименовать класс в `SoundPickerViewController` в `SoundPickerCell` (3 ссылки в одном файле)
- B) Использовать общий `Cells/SoundCell` и в SoundPicker'е тоже (но контракты разные — у Picker нет `titleLabel`)

Рекомендую A — минимальный blast radius.

## Impact

Блокирует CI/локальный билд для всех новых PR'ов. Найдено при работе над #72 — мой PR (decode failure UI banner) не может быть верифицирован build_sim'ом.

## Файлы
- `SnoozePay/SnoozePay/ViewControllers/Alarms/SoundPickerViewController.swift:92`
- `SnoozePay/SnoozePay/ViewControllers/Alarms/Cells/SoundCell.swift:7`

--- a/SnoozePay/SnoozePay/ViewControllers/Alarms/SoundPickerViewController.swift
+++ b/SnoozePay/SnoozePay/ViewControllers/Alarms/SoundPickerViewController.swift
@@ -44,7 +44,7 @@ final class SoundPickerViewController: UITableViewController {
         super.viewDidLoad()
         title = "Выбор звука"
         view.backgroundColor = .systemGroupedBackground
-        tableView.register(SoundCell.self, forCellReuseIdentifier: SoundCell.reuseID)
+        tableView.register(SoundPickerRowCell.self, forCellReuseIdentifier: SoundPickerRowCell.reuseID)
     }
 
     // MARK: - Table View
@@ -59,8 +59,8 @@ final class SoundPickerViewController: UITableViewController {
 
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         guard let cell = tableView.dequeueReusableCell(
-            withIdentifier: SoundCell.reuseID, for: indexPath
-        ) as? SoundCell else {
+            withIdentifier: SoundPickerRowCell.reuseID, for: indexPath
+        ) as? SoundPickerRowCell else {
             return UITableViewCell()
         }
 
@@ -89,9 +89,9 @@ final class SoundPickerViewController: UITableViewController {
 
 // MARK: - Sound Cell
 
-final class SoundCell: UITableViewCell {
+final class SoundPickerRowCell: UITableViewCell {
 
-    static let reuseID = "SoundCell"
+    static let reuseID = "SoundPickerRowCell"
 
     var onPlayTapped: (() -> Void)?
 

